rasa/test.py

```diff
@@ -124,7 +131,45 @@ def test_nlu(model: Optional[Text], nlu_data: Optional[Text], kwargs: Optional[D
         )
 
 
-def test_nlu_with_cross_validation(config: Text, nlu: Text, kwargs: Optional[Dict]):
+def compare_nlu_models(
+    configs: List[Text],
+    nlu: Text,
+    output: Text,
+    runs: int,
+    exclusion_percentages: List[int],
+):
+    """Trains multiple models, compares them and saves the results."""
+    from rasa.nlu.test import drop_intents_below_freq
+    from rasa.nlu.training_data import load_data
+    from rasa.nlu.utils import write_json_to_file
+    from rasa.utils.io import create_path
+    from rasa.nlu.test import compare_nlu
+    from rasa.core.test import plot_nlu_results
+
+    data = load_data(nlu)
+    data = drop_intents_below_freq(data, cutoff=5)
+
+    create_path(output)
+
+    bases = [os.path.basename(nlu_config) for nlu_config in configs]
+    model_names = [os.path.splitext(base)[0] for base in bases]
+    f_score_results = dict(
+        (model_name, [[] for _ in range(runs)]) for model_name in model_names
```

Comment:
that just appends an empty array for every run? Can do that before the dictionary comprehension to make it more readable and also less redundant? We can also us `copy()` or `runs * []` if it is modified later.